rudderstack_blueprints/verify_sync_status.py

```diff
@@ -0,0 +1,83 @@
+import sys
+import argparse
+import requests
+import shipyard_utils as shipyard
+
+
+EXIT_CODE_SYNC_STATUS_SUCCESS = 0
+EXIT_CODE_BAD_REQUEST = 201
+EXIT_CODE_SYNC_STATUS_ERROR = 210
+EXIT_CODE_SYNC_STATUS_INCOMPLETE = 211
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--access-token', dest='access_token', required=True)
+    parser.add_argument('--source-id', dest='source_id', required=False)
+    args = parser.parse_args()
+    return args
+
+
+def get_source_status(source_id, access_token):
+    access_token = "2AwmABMQwUbXLhViZfydcSDMjo1"
+    api_headers = {
+        'authorization': f"Bearer {access_token}",
+        'Content-Type': 'application/json'
+    }
+    api_url = "https://api.rudderstack.com/v2/sources"
+    source_status_url = api_url + f"/{source_id}/status"
+    source_status_json = {}
+    status_code = EXIT_CODE_SYNC_STATUS_SUCCESS
+    
+    # get response from API
+    try:
+        source_status_response = requests.get(source_status_url, 
+                                           headers=api_headers)
+        # check if successful, if not return error message
+        if source_status_response.status_code == requests.codes.ok:
+            source_status_json = source_status_response.json()
+        else:
+            print(f"Source status check failed. Reason: {source_status_response.content}")
+            sys.exit(EXIT_CODE_BAD_REQUEST)
+    except Exception as e:
+        print(f"Source {source_id} status check failed due to: {e}")
+        sys.exit(EXIT_CODE_BAD_REQUEST)
+    
+    # check source sync status
+    if source_status_json['status'] == 'finished':
+        print("Successfully managed to check sync")
+        if source_status_json.get('error'):
+            print(f"sync for sourceId: {source_id} failed with error: {source_status_json['error']}")
+            sys.exit(EXIT_CODE_SYNC_STATUS_ERROR)
+        status_code = EXIT_CODE_SYNC_STATUS_SUCCESS
```

Comment:
Do you need to set this value here, after the sys.exit? If so, shouldn't the value be EXIT_CODE_SYNC_STATUS_ERROR?